tests/runtests.py

```diff
@@ -21,7 +21,7 @@ def load_suite_tests(only=None):
             basename, ext = os.path.splitext(f)
             if (ext == '.py') and (not only_module or (only_module == basename)):
                 modname = "%s.%s" % ('.'.join(dirpath.split('/')), basename)
-                package = __import__(modname, globals(), locals(), [], -1)
+                package = __import__(modname, globals(), locals(), [], 0)
                 mod = sys.modules[modname]
```

Comment:
In Python 3 negative import levels are no longer supported (see https://docs.python.org/dev/library/functions.html#__import__). However, the import being done doesn't require a level of "-1" (even in Python 2), only level 0 so I have changed it to this.